crates/notary/server/src/server.rs

```diff
@@ -153,7 +156,16 @@ pub async fn run_server(config: &NotaryServerProperties) -> Result<(), NotarySer
         >(notary_globals.clone()))
         .route("/notarize", get(upgrade_protocol))
         .layer(CorsLayer::permissive())
-        .with_state(notary_globals);
+        .with_state(notary_globals.clone());
+
+    #[cfg(feature = "tee_quote")]
```

Comment:
This should be a simpler way, since `quote` needs to use `AuthorizationMiddleware` and doesn't need `NotaryGlobals`

```rust
// line 132
    .route(
            "/info",
            get(|| async move {
                (
                    StatusCode::OK,
                    Json(InfoResponse {
                        version,
                        public_key,
                        git_commit_hash,
                        git_commit_timestamp,
                    }),
                )
                    .into_response()
            }),
    );

    #[cfg(feature = "tee_quote")]
    {
        router = router
            .route("/quote", get(quote));
    }

    let router = router
        .route("/session", post(initialize))
        // Not applying auth middleware to ...
```